modules/store/spec/selector.spec.ts

```diff
@@ -126,13 +114,27 @@ describe('Selectors', () => {
       expect(projectFn).toHaveBeenCalledTimes(2);
     });
 
+    it('should not memoize last successful projection result in case of error', () => {
+      const firstState = { ok: 'a' };
```

Comment:
I think it would be easier to understand the logic if we would just assign `true` or `false` to `ok` here.

```suggestion
      const firstState = { ok: true };
```